scripts/scenarios/cloud6/qa-scenario-2b-sbd-kvm-vmware.yaml

```diff
@@ -131,6 +131,8 @@ proposals:
         host: vcs.qa.suse.de
         user: "@@vcenter_user@@"
         password: "@@vcenter_password@@"
+        cluster_name: []
```

Comment:
I've checked `host` vcs.qa.suse.de and there is no cluster created yet, so we have to leave this as empty or allow to pass it as parameter.